packages/gatsby-plugin-page-creator/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "gatsby-plugin-page-creator",
+  "version": "1.0.0",
+  "description": "An internal Gatsby plugin that creates pages from component files in src/pages",
+  "main": "index.js",
+  "scripts": {
+    "build": "babel src --out-dir . --ignore __tests__",
+    "watch": "babel -w src --out-dir . --ignore __tests__",
+    "prepublish": "cross-env NODE_ENV=production npm run build"
+  },
+  "keywords": [
+    "gatsby"
+  ],
+  "author": "Steven Natera <tektekpush@gmail.com>",
```

Comment:
This was an internal plugin therefore should Kyle be the author?

Reply:
yeah — though you can add yourself as a contributor (or if authors can have multiple names, add yourself there).